src/azul/plugins/metadata/hca/transform.py

```diff
@@ -82,6 +86,90 @@
 pass_thru_uuid4: PassThrough[api.UUID4] = PassThrough()
 
 
+class OrganismAge(FieldType[JSON, str]):
+
+    def to_index(self, value: JSON) -> str:
```

Comment:
Please post deep link to the part of the Coveralls report that proves that this doctest is run on Travis.

Reply:
[Here](https://codecov.io/gh/DataBiosphere/azul/pull/1909/diff#D7-91). I missed to add the module to the `test_doctests.py` file in the previous review.